zksync_sdk/wallet.py

```diff
@@ -100,14 +99,13 @@ async def set_signing_key(self, fee_token: TokenLike, *,
     # This function takes as a parameter the integer fee of 
     # lowest token denominations (wei, satoshi, etc.)
     async def build_change_pub_key(
-        self, 
-        fee_token: Token,
-        eth_auth_data: Union[ChangePubKeyCREATE2, ChangePubKeyEcdsa, None],
-        fee: int,
-        nonce: Optional[int] = None,
-        valid_from=DEFAULT_VALID_FROM, 
-        valid_until=DEFAULT_VALID_UNTIL
-    ):
+            self,
+            fee_token: Token,
+            eth_auth_data: Union[ChangePubKeyCREATE2, ChangePubKeyEcdsa, None],
+            fee: int,
+            nonce: int = None,
```

Comment:
The optional should've remained here, right? It seems like  a part of the typing introduced by the PR

Reply:
I should work like even without it. But yes, you are right. How can I checkout to this branch and fix it by the commit ?

Reply:
I guess the best way is to start the new branch which preserves the history with @hukkin's commits and push it separately

Reply:
Do you mean checkout the : hukkin/zksync-python repo create the separated branch inside the master and do the new PR for that rigtht?

Reply:
I've fixed mypy in the current branch for you

Reply:
Implicit optional is not PEP484 compliant and mypy will error on it in the future: https://github.com/python/mypy/issues/9091

Reply:
But It's not all. I'm doing the fixes now. there is only 1 question: Can i push into this repo separated branch, that's it?

Reply:
If you have write access to zksync-sdk/zksync-python you can also push to my branch

Reply:
There must be 1 more thing: 
transfer, eth_signature = await self.build_forced_exit(target, `token_obj`, fee_int 
instead of just token

Reply:
yeah that's fixed in the commit I made